pages/FAQ/_index.md

```diff
@@ -117,6 +117,7 @@ update the package. Paru has been problematic with updating before, use Yay.
 ### How do I screen lock?
 
 Use a Wayland-compatible locking utility using WLR protocols, e.g. `swaylock`.
+Be aware that they will not prevent tty-changing using Ctrl-Alt-F3.
```

Comment:
Greeters usually start on tty1 or tty7, so any `Ctrl-Alt-F2..6` combo works. Maybe we should drop the keychord?

Reply:
I changed it to Ctrl-Alt-F1...F7